src/config/wmodules-monitor.c

```diff
@@ -10,14 +10,17 @@
  */
 
 #ifndef CLIENT
-#include "wazuh_modules/wmodules.h"
+#include "../wazuh_modules/wmodules.h"
 #include "../monitord/monitord.h"
 
 // Parse XML configuration
-int wm_monitor_read(const OS_XML *xml, XML_NODE node, wmodule *module) {
+int wm_monitor_read(__attribute__((unused)) const OS_XML *xml, __attribute__((unused)) XML_NODE node, wmodule *module) {
     if (module) {
         module->context = &WM_MONITOR_CONTEXT;
         module->tag = strdup(module->context->name);
```

Comment:
this should be os_strdup to avoid coverity issues.

Reply:
Agree! I'll change it.